### ISSUE TITLE
Looking for new maintainer(s) !

### ISSUE THREAD

**Issue by JocelynDelalande** [oasiswork/redmine-gitlab-migrator#20]
I have been shortly maintaining this project. But as I no longer use it, I don't and I won't maintain it. Neither @oasiswork/owners will, I think. That is why nothing happens on master since 2 years, despite the fact there seems to be small but renewed interest for the project (watch/issues/PR).

So what I suggest:

- someone (ideally several people) start maintaining this project **Who's in ?**
- We hand off to that people
  - github access
  - travis access
  - pip access

The project could be moved to a new org/person or left here, depending on new maintainers and @oasiswork/owners (@zacbri ?) wills.

So… **who's in ?**

cc @johanv @gyscos @scambra

**Comment by JocelynDelalande** [comment]
Also cc @tuxem 

For the record, I discovered tha I no longer have maintainers rights on the repo, but maybe @tuxem or @zacbri could help to hand-off maintainership if someone pops in :).

**Comment by scambra** [comment]
I could maintain this project, but it would be nice if someone else help me
I'm going to use this project to migrate few projects from redmine to gitlab.com, but afterthat I will be in same situation as @JocelynDelalande, I will no longer use it, and I wouldn't like to be only maintainer or probably will stop maintain it.

I guess everyone can be in same situation, because of project nature, most people will use few times after migrating everything.

About moving to new org/person, I think it may stay in @oasiswork org, or create new one org, so it isn't moved anymore if maintainer change again

**Comment by JocelynDelalande** [comment]
@zacbri @tuxem Would it be possible to transfer the project to the newly created org https://github.com/orgs/redmine-gitlab-migrator so that the project can continue to live ?

Thanks a lot :-)

Jocelyn

**Comment by JocelynDelalande** [comment]
@scambra good thinking. I acted accordingly to your suggestion, trying to be future-proof and hoping an answer to my previous comment :)

**Comment by scambra** [comment]
I joined to new org, waiting for repo transfer

**Comment by zacbri** [comment]
Repo transfered.
Thanks @scambra for taking over. Enjoy !

**Comment by JocelynDelalande** [comment]
@scambra Great, I saw that you resumed activity on the project :-)

**Comment by scambra** [comment]
Yes, I merged all commits in my clone, some of them taken from other forks

**Comment by romanr** [comment]
Is this project abandoned? Or if it is transferred to new maintainer then why this issue is open?

**Comment by scambra** [comment]
Is it transferred, new maintainer is redmine-gitlab-migrator group, 
It's still open because it would be nice to get more people willing to help maintaining, e.g. some forks' owners

**Comment by thomas955** [comment]
Im would help for now. Currently i have to migrate some projects to gitlab and already fixed some errors regarding tests, and api changes from gitlab api v3 to v4.

**Comment by JocelynDelalande** [comment]
@thomas955 happy to read that, thanks :-).

 I let @scambra see with you how your collaboration could go and optionally granting you accesses.

**Comment by scambra** [comment]
@thomas955 I added you as collaborator

**Comment by AdamN1** [comment]
Hi,
if there is a need we can help keep the code.

**Comment by scambra** [comment]
@AdamN1, do you belong to @JustNet?

**Comment by AdamN1** [comment]
> @AdamN1, do you belong to @JustNet?

Yes, I'm PM

**Comment by scambra** [comment]
@AdamN1 I have added you as owner, so you can add other people from your organization.
